src/Menu/Menu.js

```diff
@@ -396,10 +399,27 @@ class Menu extends Component {
         event.preventDefault();
         this.decrementKeyboardFocusIndex();
         break;
+      default:
+        const hotKeys = this.hotKeyHolder.append(key);
+        if (this.setFocusIndexStartsWith(hotKeys)) {
+          event.preventDefault();
+        }
     }
     this.props.onKeyDown(event);
   };
 
+  setFocusIndexStartsWith(keys) {
+    const found = this.props.children.findIndex((child) => {
```

Comment:
`this.props.children` can be something else than an array. We should use a defensive approach. Have a look at https://facebook.github.io/react/docs/top-level-api.html.